recipes/gtest/all/test_package/CMakeLists.txt

```diff
@@ -4,13 +4,15 @@ project(test_package CXX)
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup()
 
+find_package(GTest REQUIRED)
+
 if(WITH_MAIN)
     add_executable(${PROJECT_NAME} test_package.cpp)
 else()
     add_executable(${PROJECT_NAME} main.cpp test_package.cpp)
 endif()
 
-target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
+target_link_libraries(${PROJECT_NAME} GTest::gtest GTest::gtest_main GTest::gmock GTest::gmock_main)
```

Comment:
Both main and gmock are optional, you have to add conditionals, otherwise, it will break when they are not available.

Reply:
done